src/cluster.c

```diff
@@ -5327,7 +5331,9 @@ void restoreCommand(client *c) {
     }
 
     /* Verify RDB version and data checksum. */
-    if (verifyDumpPayload(c->argv[3]->ptr,sdslen(c->argv[3]->ptr)) == C_ERR)
+    int rdbver;
```

Comment:
rdbver is unused

Reply:
i think we can revert the change in verifyDumpPayload

Reply:
Yep, forgot that leftover. Resolved via 5165321a4